ckanext/datastore/controller.py

```diff
@@ -12,6 +13,20 @@
 from ckan.common import request
 
 
+def _dump_nested(column, record):
+    name, ctype = column
+    value = record[name]
+
+    is_nested = (
```

Comment:
I believe the final logic was changed in #3487: https://github.com/ckan/ckan/pull/3487/files#diff-8585ec6c5af023a7b43945d3ad6caeeaR13